cygwin.rst

```diff
@@ -0,0 +1,194 @@
+*****************************
+Docs in Windows (Cygwin) 
+*****************************
+
+.. _cygwin-docs:
+
+Cygwin
+-------------------------------
+Cygwin is windows implementation of unix-like commands. You can basically use many linux commands from windows 
+command propmpt.
+
+Install `cygwin <https://cygwin.com/install.html/>`_ and add its path into windows environmental variable.
+This will allow you to `use cygwin from cmd promopt <https://www.howtogeek.com/howto/41382/how-to-use-linux-commands-in-windows-with-cygwin/>`_.
+
+
+.. _python:
+
+Python 3
+-------------------------------
+You need to install `python 3 <https://www.python.org/downloads/>`_.
+During the installation, you can select to set up pip tool as well as add python path to the Path environment variable.
+
+
+
+.. _virtualenv:
+
+Virtual Environment
+-------------------------------
+Virtual environment is a useful tool particularly if you work with multiple projects, each requires different dependencies and python versions. By using virtualenv, you can create seperated virtual environment and install packages as you need without affecting the main 
+python installation.
+
+To install virtualenv in windows, you can basically use pip command, which was already installed with python 3.6.2.
+
+.. code-block:: rest
+
+  $ pip install virtualenv
+
+.. note::
+  You need to make sure you have already installed pip in the previous step.
+
+Create a new directory for your odkdocs work:
+
+.. code-block:: rest
+
+  $ mkdir odk
+    
+You have two options: 
+
+  - Use the native virtualenv.
+  - Use virtualenvwrapper.
+
+
+.. _native-virenv:
+
+Native virtualenv
+~~~~~~~~~~~~~~~~~~~~~
+Create a new python 3 virtual environment, "odkenv" is the name of the virtualenv, you are free to add your own.
+
+.. code-block:: rest
+
+  $ virtualenv --python=<python path/python.exe> odkenv
+ 
+After creating python3 virtualenv in the previous step, multiple files are coppied into the folder odkenv.
+
+.. code-block:: rest
+
+  $ ls odkenv
+
+The folder Scripts contains all virtualenv control as bat files.
+
+To activate the odkenv:
+
+.. code-block:: rest
```

Comment:
This progression of commands doesn't make literal sense. If I `cd` into something, I don't need to write the full path to it to execute.

Reply:
In windows, you need to move into the full path of a program to execute it. Otherwise, you need to add its full path into windows environment Path.

Reply:
YEs, but my point was- if you did CD into it, you don't also need to have the full path, which it looked like you had.

Reply:
I will check that. Thank you :)